swap/swap_test.go

```diff
@@ -125,7 +125,7 @@ func TestGetAllBalances(t *testing.T) {
 }
 
 func testBalances(t *testing.T, swap *Swap, expectedBalances map[enode.ID]int64) {
-	balances := swap.GetAllBalances()
+	balances, _ := swap.GetAllBalances()
```

Comment:
Let's not ignore the error coming back, even if it is trivial here?

Reply:
You're absolutely right, fixed.

I also changed the code so that we are no longer ignoring errors in the `swarm/Balance` and `swarm/Balances` functions; they are now returned.